packages/core/src/browser/status-bar/status-bar.tsx

```diff
@@ -221,7 +228,7 @@ export class StatusBarImpl extends ReactWidget implements StatusBar {
         });
         const elementInnerDiv = <React.Fragment>{children}</React.Fragment>;
 
-        return React.createElement('div', { key: id, ...this.createAttributes(entry) }, elementInnerDiv);
+        return React.createElement('div', { key: id, ...this.createAttributes(entry), 'aria-label': entry.ariaLabel }, elementInnerDiv);
```

Comment:
I wonder whether it wouldn't be simpler to fall back to `'aria-label: entry.text'` rather then setting it in various places as follows:
```suggestion
        return React.createElement('div', { key: id, ...this.createAttributes(entry), 'aria-label': entry.ariaLabel ?? entry.text }, elementInnerDiv);
```